README.md

```diff
@@ -74,39 +60,17 @@ CLI. For more information on this, please see this [blog post](https://amartin.c
 ### [lazy.nvim](https://github.com/folke/lazy.nvim)
 
 ```lua
-{
-  "vhyrro/luarocks.nvim",
-  priority = 1000,
-  config = true,
-  opts = {
-    rocks = { "lua-curl", "nvim-nio", "mimetypes", "xml2lua" }
-  }
-},
 {
   "rest-nvim/rest.nvim",
-  ft = "http",
-  dependencies = { "luarocks.nvim" },
-  config = function()
-    require("rest-nvim").setup()
-  end,
 }
 ```
 
-> [!NOTE]
->
-> There's a `build.lua` file in the repository that `lazy.nvim` will find and source to install the
-> luarocks dependencies for you by using `luarocks.nvim`. You don't need to specify a rock list
-> by yourself.
-
 ### [packer.nvim](https://github.com/wbthomason/packer.nvim)
 
 ```lua
 use {
   "rest-nvim/rest.nvim",
-  rocks = { "lua-curl", "nvim-nio", "mimetypes", "xml2lua" },
-  config = function()
-    require("rest-nvim").setup()
-  end,
+  rocks = { "nvim-nio", "mimetypes", "xml2lua", "fidget.nvim" },
```

Comment:
Wondering why we don't need these dependencies when installing by `lazy.nvim`?

Reply:
afaik lazy.nvim now supports luarocks dependencies, so as `rest-nvim` is on luarocks, lazy.nvim can know all needed dependencies.
For example, [`neorg` also don't need to write dependencies when installing by `lazy.nvim`](https://github.com/nvim-neorg/neorg?tab=readme-ov-file#lazynvim)